example/storybook/package.json

```diff
@@ -23,7 +23,7 @@
     "@dank-style/animation-plugin": "^0.0.4",
     "@dank-style/react": "^0.3.10",
     "@expo/html-elements": "^0.4.1",
-    "@gluestack/design-system": "0.3.35",
+    "@gluestack/design-system": "^0.3.44",
```

Comment:
Upgrade the package to its latest.

Reply:
fixed : updated to latest version to 0.3.45